src/quic/sessionticket.cc

```diff
@@ -0,0 +1,177 @@
+#if HAVE_OPENSSL && NODE_OPENSSL_HAS_QUIC
+
+#include <env-inl.h>
+#include <memory_tracker-inl.h>
+#include <node_errors.h>
+#include <ngtcp2/ngtcp2_crypto.h>
+#include <node_buffer.h>
+#include "sessionticket.h"
```

Comment:
```suggestion
#include "sessionticket.h"
#include <env-inl.h>
#include <memory_tracker-inl.h>
#include <ngtcp2/ngtcp2_crypto.h>
#include <node_buffer.h>
#include <node_errors.h>
```
To make the linter happy.